src/formatter.rs

```diff
@@ -88,6 +89,32 @@ where
                 serializer.serialize_entry("level", level)?;
             }
 
+            #[cfg(feature = "ansi_logs")]
+            {
+                let level = match *metadata.level() {
```

Comment:
it would be nice if this was restructured a bit to use the same conversion from level => string, so that if we ever change it, it would automatically be changed in both place